Google.Cloud.EntityFrameworkCore.Spanner/Storage/Internal/SpannerRelationalConnection.cs

```diff
@@ -95,14 +95,10 @@ protected override DbConnection CreateDbConnection()
         /// </summary>
         public ISpannerRelationalConnection CreateMasterConnection()
         {
-            var builder = new SpannerConnectionStringBuilder(ConnectionString);
```

Comment:
This would only reuse the configuration that was set in the `ConnectionString`, and not any additional configuration that might have been set in the `SessionPoolManager` and/or `ChannelCredentials`.